models/decision.js

```diff
@@ -0,0 +1,38 @@
+import {analyse} from '@lblod/marawa/rdfa-context-scanner';
+import {cleanupTriples} from '../support/pre-importer';
+import { editorDocumentFromUuid } from '../support/editor-document';
+
+export default class Decision {
+  // TODO: perhaps not the best name or place, given that it returns an array
```

Comment:
Maybe a helper function like extractDecisionsFromDocument is better?

Reply:
yeah that name sounds good
could stay as a static function here, don't think it's too out of place